mgmt/ProxyConfig.h

```diff
@@ -98,15 +98,31 @@ class ConfigProcessor
 
 // A Continuation wrapper that calls the static reconfigure() method of the given class.
 template <typename UpdateClass> struct ConfigUpdateContinuation : public Continuation {
+  void (*f)();
   int
   update(int /* etype */, void * /* data */)
   {
     UpdateClass::reconfigure();
     delete this;
     return EVENT_DONE;
   }
+  int
+  update_func(int /*etype*/, void * /* data */)
```

Comment:
i'm not grok'ing most of this code. It seems the intent of this template was to always use the CB named ::reconfigure() from the templated class. That doesn't work for this use case?